app/models/chargeback/report_options.rb

```diff
@@ -0,0 +1,102 @@
+class Chargeback
```

Comment:
This just adds one identifier (`ReportOptions`) to the namespace of Chargeback class.

I think there are other places in MiQ. using the same feature.